.github/actions/spell-check/patterns/patterns.txt

```diff
@@ -1,8 +1,10 @@
 https://(?:(?:[-a-zA-Z0-9?&=]*\.|)microsoft\.com)/[-a-zA-Z0-9?&=_\/.]*
 https://aka\.ms/[-a-zA-Z0-9?&=\/_]*
+https://www\.vt100\.net/docs/[-a-zA-Z0-9#_\/.]*
+http://www\.itscj\.ipsj\.or\.jp/iso-ir/[-0-9]+\.pdf
```

Comment:
```suggestion
https://www\.itscj\.ipsj\.or\.jp/iso-ir/[-0-9]+\.pdf
```

Reply:
You're absolutely right. I have no idea why I didn't do this. I've pushed up an update with those fixed now. Thanks.